exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/cloud/01_import_data.ipynb

```diff
@@ -2,41 +2,23 @@
  "cells": [
```

Comment:
<p>The cloud_store_config.ipynb runs the general config nb. This is not always necessary since the configuration page shall be visited just once, unless the customer wants to change the datasource. I suggest we do this notebook it in the same fashion as all other notebooks. All others start from the prerequisites, i.e. "Please make sure that such and such steps are completed before running this notebook". This is a subject of a another ticket of course.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Ah, I thought our configuration detects multiple runs automatically and not showing configuration interface when initial setup was done already (might be good addition to the configuration mechanisms)</p>